documentacion/como_empezar/README.md

```diff
@@ -62,7 +62,7 @@ gem install transbank-sdk
 
 Te recomendamos leer [las instrucciones de instalación detalladas para el SDK Ruby](https://github.com/TransbankDevelopers/transbank-sdk-ruby#instalaci%C3%B3n) para mas opciones de instalación.
 
-(Para webpay en Ruby puedes seguir usando [libwebpay](https://github.com/TransbankDevelopers/libwebpay-ruby) u otra alternativa)
+(Para Webpay en Ruby puedes seguir usando [libwebpay](https://github.com/TransbankDevelopers/libwebpay-ruby) u otra alternativa)
 
 [En
```

Comment:
En [**Python**](link)...

Reply:
sorry the link is broken, can you please elaborate more?